modules/sched/type.go

```diff
@@ -0,0 +1,77 @@
+//go:build sched
+// +build sched
+
+package sched
+
+import (
+	"context"
+
+	"codnect.io/chrono"
+	"github.com/risor-io/risor/errz"
+	"github.com/risor-io/risor/object"
+	"github.com/risor-io/risor/op"
+)
+
+type task struct {
+	Name string
+	t    chrono.ScheduledTask
+}
+
+// IsTruthy returns true if the task is not cancelled.
+func (t *task) IsTruthy() bool {
+	return !t.t.IsCancelled()
+}
+
+// Cost returns the cost of the task.
+func (t *task) Cost() int {
+	return 0
+}
+
+// Equals returns true if the task is equal to the other object.
+func (t *task) Equals(other object.Object) object.Object {
+	so, ok := other.(*task)
+	if !ok {
+		return object.False
+	}
+	ok = (*t == *so)
+	return object.NewBool(ok)
+}
+
+// Inspect returns the string representation of the task.
+func (t *task) Inspect() string {
+	return "sched.task"
+}
+
+// Type returns the type of the task.
+func (t *task) Type() object.Type {
+	return object.Type("sched.task")
+}
+
+func (t *task) Interface() any {
+	return t
```

Comment:
Little thing: usually `Interface` returns the Go type that is being wrapped, so `t.t` (the `chrono.ScheduledTask`) in this case.

Reply:
Fixed!